eng/pipelines/check-base-image-updates.yml

```
@@ -27,7 +27,8 @@ extends:
     customBuildTags:
     - ES365AIMigrationTooling
     stages:
-    - stage: Check Base Images
+    - stage:
```

```suggestion
    - stage: CheckBaseImages
```

I haven't seen any documentation that suggests an empty stage or job name will be accepted.

---

FWIW the pipeline works without one.

---

You learn something new every day.